rust/src/main.rs

```diff
@@ -3,7 +3,62 @@ mod private;
 
 use std::error::Error;
 
+async fn async_main() {
+    match public::get_server_time_async().await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_markets_async().await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_products_async().await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_assets_async().await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_assets_crypto_async().await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_assets_fiat_async().await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_asset_pairs_async("crypto","spot","ID").await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_ticker_async("btc_usdt_spot").await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_trades_async("2", "btc_usdt_spot").await {
+        Err(e) => println!("{:?}", e),
+        _ => ()
+    }
+    match public::get_depth_async("btc_usdt_spot", "5").await {
```

Comment:
don't hardcode the symbol, use the parameter instrument

Reply:
Sorry, I don't understand, can you please elaborate?

Reply:
> I think what mas dendi means is you put all the instrument symbols in config.rs just like in babingepet repo, so to use the symbol we don't harcode it directly. CMIIW mas den

Reply:
Oh I get it, this is what you meant right? https://github.com/bitwyre/sdk/blob/dbf08f47d71b8ff67e6ccc52d2e2119793120ca6/rust/src/main.rs#L16

Reply:
I resolved this, can you please re-review? @dendisuhubdy